src/index.html

```diff
@@ -7,8 +7,42 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="http://127.0.0.1:5500/src/index.html" class="logo"><img src="images/logo.png" alt="images"></a>
```

Comment:
```suggestion
      <a href="#" class="logo">
        <img
          src="images/logo.png"
          alt="images"
        >
      </a>
```
не потрібно прописувати такий атрибут href